Dockerfile.main

```diff
@@ -6,5 +6,6 @@ RUN source /opt/lsst/software/stack/loadLSST.bash \
         flask \
         gunicorn \
         python-confluent-kafka \
+        openssl=3.1.4 \
```

Comment:
Would it be possible to add a comment here that this is a workaround for DM-XXXXX?

I'm not familiar with mamba, so I don't know how this line will interact with Conda's `rubin-env`. I think it would be good to test this before merging.